compile.sh

```diff
@@ -37,7 +36,24 @@ if [ "$_has_dtbo" = "true" ]; then
 fi
 
 _self_dir=$(realpath $(dirname "$0"))
-. $_self_dir/compile_$_compiler.sh
+. $_self_dir/setup_$_compiler.sh
+_cross_compile_32="$ANDROID_ROOT/prebuilts/gcc/linux-x86/arm/arm-linux-androideabi-4.9/bin/arm-linux-androideabi-"
+_build_cmd="make O=$_out ARCH=arm64 -j$(nproc) $_make_args CROSS_COMPILE_ARM32=$_cross_compile_32"
```

Comment:
What about:

```
make_args+=" CROSS_COMPILE_ARM32=$_cross_compile_32"
_build_cmd="make $_make_vars $_make_args"
```

without removing the `_make_vars `variable assignment in line 18? This way we keep that variable to "tune" and keep separated the args and vars.

Reply:
Well there's no clear distinction between vars and args so I've concatenated them all to the same variable. Let me know what you think.